.gitmodules

```diff
@@ -19,3 +16,6 @@
 [submodule "modules/libcanard"]
 	path = modules/libcanard
 	url = git://github.com/ArduPilot/libcanard.git
+[submodule "modules/waf"]
+	path = modules/waf
+	url = https://gitlab.com/ita1024/waf.git
```

Comment:
uh, I am pretty sure we don't want to clone waf master branch !

Reply:
Just a test to check something. Won't be in the final PR